tests/test_app.py

```diff
@@ -350,6 +358,7 @@ def test_ping_proposal(self):
         self.assertEqual({}, re.json)
 
     def test_node_send_stats_with_unknown_node(self):
+        settings.RESTRICT_BY_IP_ENABLED = False
```

Comment:
All these disabling wouldn't be required, if this feature was disabled by default.

Reply:
removed